lintreview/utils.py

```diff
@@ -19,26 +19,24 @@ def in_path(name):
     return False
 
 
-def npm_exists(name):
+def npm_exists(name, cwd=os.getcwd()):
```

Comment:
This default value will capture the cwd() when the module is imported. I don't think that's what you want.

Reply:
I must not fully understand how it works, then. Could you explain more? I could use None as default and set to os.getcwd() if None is passed.

Reply:
The current default value will be created when this module is imported, not each time the function is run. Using `None` and getting cwd if none is passed will work as you want.

Reply:
I see, thank you.